maldoca/ole/vba_extract_test.cc

```diff
@@ -68,7 +68,7 @@
 #include "gtest/gtest.h"
```

Comment:
what was changed in "maldoca/ole/testdata/ole/c98661bcd5bd2e5df06d3432890e7a2e8d6a3edcb5f89f6aaa2e5c79d4619f3d"? (change above)

Reply:
put it back... arrgh windows defender!!!